src/components/MyAccount/Settings/AccountSettingsUtils.ts

```diff
@@ -73,7 +65,9 @@ export const parsePayload = (formSubmissionBody, settingsData: Patron) => {
   return accountSettings.reduce((putRequestPayload, setting) => {
```

Comment:
`parseSettingsPayload` would make more sense to me here, in general I think the function/var names in this file could be more specific

Reply:
updated